packages/blade/src/components/Indicator/Indicator.tsx

```diff
@@ -88,21 +88,24 @@ const Indicator = ({
   }, [size]);
   const dimensions = getDimension();
 
-  const isReactNative = getPlatformType() === 'react-native';
-  const isWeb = !isReactNative;
+  const isWeb = !isReactNative();
   const a11yProps = makeAccessible({
     label: accessibilityLabel ?? childrenString,
     ...(isWeb && { role: 'status' }),
   });
 
   return (
     <BaseBox
-      display="flex"
+      display={(isWeb ? 'inline-flex' : 'flex') as never}
       {...a11yProps}
       {...metaAttribute({ name: MetaConstants.Indicator, testID })}
       {...getStyledProps(styledProps)}
     >
-      <BaseBox display="flex" flexDirection="row" alignItems="center">
+      <BaseBox
+        display={(isWeb ? 'inline-flex' : 'flex') as never}
```

Comment:
we don't have to make inner item `inline-` right? if outer item is inner-flex the inner item will anyway not extend the width 🤔

Reply:
@saurabhdaware I added it since `Badge` and `Counter` have it as well. I feel it should not cause any issue

Reply:
I think we should remove it from Badge and Counter instead. It was incorrectly fixed earlier. 

Won't cause an issue but we have to add that extra check everywhere so better to remove it if its not needed

Reply:
I agree, will remove it.